anndata_dask.py

```diff
@@ -146,9 +146,11 @@ def mk_dataframe_view(sub, ann, key):
         def mk_dict_view(dat, ann, key):
             return DictView(dat, view_args=(ann, key))
 
-        self._obs = daskify_call(mk_dataframe_view, obs_sub, self, "obs")
-        self._var = daskify_call(mk_dataframe_view, var_sub, self, "var")
-        self._uns = daskify_call(mk_dataframe_view, uns_new, self, "uns")
+        self._obs = daskify_call_return_df(mk_dataframe_view, obs_sub, self, "obs",
+                                           _dask_meta=obs_sub._meta)
+        self._var = daskify_call_return_df(mk_dataframe_view, var_sub, self, "var",
+                                           _dask_meta=var_sub._meta)
```

Comment:
Previously `._obs` and `._var` were DDFs when initially constructed, but became opaque `Delayed` objects when a view of the original was created (by `ad_view = ad_regular[expr]`, etc.)